src/InvalidArgumentException.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Kodus\Cache;
+
+class InvalidArgumentException extends \InvalidArgumentException implements \Psr\SimpleCache\InvalidArgumentException
+{
```

Comment:
shouldn't we import the extended and implemented classes instead of using the full namespace ?